lib/api/blocks.js

```diff
@@ -206,7 +252,7 @@ module.exports = function (app) {
 					if (err) {
 						return error({ success: false, error: err });
 					}
-					return success(result);
+					return success(_.extend({ success: true }, result));
```

Comment:
why `_.extend`?

Reply:
Again, we combine the status with the actual result.